src/format/parse.rs

```diff
@@ -564,28 +564,26 @@ mod tests {
     use crate::format::*;
     use crate::{DateTime, FixedOffset, TimeZone, Timelike, Utc};
 
-    #[test]
-    fn test_parse() {
-        use crate::format::InternalInternal::*;
-        use crate::format::Item::{Literal, Space};
-        use crate::format::Numeric::*;
+    fn parse_all(s: &str, items: &[Item]) -> ParseResult<Parsed> {
```

Comment:
Style nit: helper below the tests, please.

Reply:
I'll learn at some point :smile: